build.fsx

```diff
@@ -55,7 +55,7 @@ let runIncrementally = hasBuildParam "incremental"
 let incrementalistReport = output @@ "incrementalist.txt"
 
 // Configuration values for tests
-let testNetFrameworkVersion = "net471"
+let testNetFrameworkVersion = "net48"
```

Comment:
Upgraded to .NET 4.8 so we could use newer versions of Verify, which had `DateTime` scrubbing. I ended up not needing it but I don't think it hurts anything given that this only affects the test suite.